TreeTracker/Resources/Localizable.strings

```diff
@@ -99,3 +100,25 @@
 "Settings.PhotoQuality.Option.Medium.Info" = "Medium resolution, medium data usage.";
 "Settings.PhotoQuality.Option.High.Title" = "High";
 "Settings.PhotoQuality.Option.High.Info" = "High resolution, high data usage.";
+
+//ChatList Screen
+"ChatList.SyncMessages.normal.Title" = "Sync Messages";
```

Comment:
Nit
```suggestion
"ChatList.SyncMessages.normal.Title" = "Update Messages";
```

Reply:
As a dev the action is sync but as a user, they understand `Refresh` or `update`.